plugin-server/tests/main/process-event.test.ts

```diff
@@ -1032,6 +1032,116 @@ test('snapshot event stored as session_recording_event', async () => {
     expect(event.snapshot_data).toEqual({ timestamp: 123 })
 })
 
+test('performance event stored as performance_event', async () => {
+    await eventsProcessor.processEvent(
+        'some-id',
+        '',
+        {
+            event: '$performance_event',
```

Comment:
@benjackwhite this was mis-spelled `perfomance` not `performance` 🤣 

I could run the tests by `pnpm setup:test` and then `pnpm test -- -i tests/main/process-event.test.ts -t 'performance event stored as performance_event'`.